### PR TITLE
explicit directions for using summon with summon-conjur

### PR DESCRIPTION
**Pull request by doodlesbykumbi**
Addresses https://github.com/cyberark/summon-conjur/issues/14 https://github.com/cyberark/summon-conjur/issues/15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,53 @@
 # summon-conjur
 
-Conjur provider for [Summon](https://conjurinc.github.io/summon/).
+Conjur provider for [Summon](https://cyberark.github.io/summon/).
```
Comment (ghost) [inline README.md]: Can you switch this to the normal GitHub repo link? Not the docs link. 

TBD on what ^ site will become.

**Review by ghost — CHANGES_REQUESTED** [review]
One link change and we're good to merge. Thanks for taking these docs on Kumbi :)

**Comment by doodlesbykumbi** [comment]
@dustinmm80 link changed :)

**Review by ghost — APPROVED** [review]
Merging

**Pull request merged**